daft/subscribers/dashboard.py

```diff
@@ -70,7 +64,7 @@ def broadcast_query_information(
     data = json.dumps(
         {
             "id": str(uuid.uuid4()),
-            "optimized_plan": optimized_plan,
+            "optimized_plan": unoptimized_plan,
```

Comment:
This is just a temporary change, next PR will turn the dashboard into a query subscriber and we will fix it.